marketplace/deployer_util/provision_test.py

```diff
@@ -43,3 +46,17 @@ def test_limit_name(self):
   def assertModifiedName(self, text, expected):
     self.assertEqual(text[:-5], expected)
     self.assertRegexpMatches(text[-5:], r'-[a-f0-9]{4}')
+
+  def test_deployer_image_inject(self):
+    schema = config_helper.Schema.load_yaml('''
+    properties:
+      deployer_image:
+        type: string
+        x-google-marketplace:
+          type: DEPLOYER_IMAGE
+    ''')
+    values = {}
+    deployer_image = 'gcr.io/cloud-marketplace/partner/solution/deployer:latest'
+    self.assertEquals(
+        provision.inject_deployer_image_properties(
```

Comment:
removing the return value will inconvenience the test--well I'm not opposed to having the method returning the input `values` for this purpose